src/SFA.DAS.AssessorService.Web.UnitTests/Validators/Standard/OptOutStandardConfirmViewModelValidatorTests.cs

```diff
@@ -17,7 +17,7 @@ namespace SFA.DAS.AssessorService.Web.UnitTests.Validators.Standard;
 
 public class OptOutStandardConfirmViewModelValidatorTests
 {
-    [Test, RecursiveMoqAutoData]
+    [Test, MoqAutoData]
```

Comment:
This should be combined test set with this existing rule and the new one when the 2 validators have been combined.

There is a missing test class for the Optin validator

Reply:
Combined and added tests for the `OptInStandardVersionViewModel`